src/wikmd/wiki.py

```diff
@@ -470,11 +470,11 @@ def nav_id_to_page(id):
     return redirect("/")
 
 
-@app.route(os.path.join("/", cfg.images_route, "<path:image_name>"))
+@app.route(f"/{cfg.images_route}/<path:image_name>")
```

Comment:
This two fixes in combination fixes this error you mentioned
> Image upload works. But the uploaded image can not be viewed. Image lookup is done in src folder.

However, joining the path like this for the route isn't safe as if the `cfg.image_route` starts or ends with a `/` there will be two in a row. Of course, someone shouldn't define a route like that but could trip someone up.

If we changed the joining too something like  `(Path("/") / cfg.images_route / "<path:image_name>").as_posix()` this would take that into account but it is a lot less readable.

Another thing to note is - why would someone need to change the image upload route?

Reply:
> However, joining the path like this for the route isn't safe as if the cfg.image_route starts or ends with a / there will be two in a row. Of course, someone shouldn't define a route like that but could trip someone up.

I think we can keep using this for now.

> Another thing to note is - why would someone need to change the image upload route?

This is just to be able to support existing wiki's, who's image system has an other structure.